### PR TITLE
#ADP add deploy script

### PR DESCRIPTION
**Pull request by ddimitrioglo**

### CODE REVIEW

File: bin/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```
Comment (AlexanderC) [inline bin/deploy.sh]: - [ ] Please add binaries check `if [ -z $(which aws) ]; then ... fi`
- [ ] Please move aws profile (e.g. `--profile default`) and region (e.g. `--region eu-central-1`)

File: package.json [hunk 2]
```diff
@@ -7,9 +7,10 @@
   "license": "MIT",
   "scripts": {
     "start": "node ./bin/server",
+    "build-assets": "gulp build-assets",
     "start-debug": "DEBUG=atm-docs:* node ./bin/server",
-    "build": "gulp compile-pages && gulp build-assets",
-    "watcher": "gulp sass:watch"
+    "watcher": "gulp sass:watch",
+    "deploy": "sh ./bin/deploy.sh"
```
Comment (AlexanderC) [inline package.json]: Please use short invocation definition `./bin/deploy.sh`

File: package.json [hunk 3]
```diff
@@ -7,9 +7,10 @@
   "license": "MIT",
   "scripts": {
     "start": "node ./bin/server",
+    "build-assets": "gulp build-assets",
```
Comment (AlexanderC) [inline package.json]: Please move `gulp` to local dependencies (in `devDependencies`) and use local binaries instead (e.g. `./node_modules/.bin/gulp`)

File: bin/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+echo 'Compiling pages'
+gulp compile-pages
+
+echo 'Compiling assets'
+gulp build-assets
```
Comment (AlexanderC) [inline bin/deploy.sh]: Please use `npm run build-assets` command

File: bin/deploy.sh [hunk 5]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo 'Compiling pages'
+gulp compile-pages
```
Comment (AlexanderC) [inline bin/deploy.sh]: Please configure this similar to `npm run build-assets`

File: bin/deploy.sh (hunk 1 above)
Comment (AlexanderC) [inline bin/deploy.sh]: Note that error exit code is `exit 1`

Comment (ddimitrioglo) [inline bin/deploy.sh]: done

**Pull request merged**